rotkehlchen/globaldb/handler.py

```diff
@@ -344,11 +344,18 @@ def search_assets_levenshtein(
                 if entry[1] is None and entry[2] is None:
                     continue
 
-                lev_dist_name = levenshtein(substring_search, entry[1].lower()) if entry[1] is not None else 0  # noqa: E501
-                lev_dist_symbol = levenshtein(substring_search, entry[2].lower()) if entry[2] is not None else 0  # noqa: E501
+                lev_dist_name = levenshtein(substring_search, entry[1].casefold()) if entry[1] is not None else 0  # noqa: E501
```

Comment:
you are doing entry[1] and entry[2] casefold, two different times. Here and below. Precompute it once and use it in both places

Reply:
because `entry[1]` or `entry[2]` can be `None`. I can only precompute the tuple lookup